package.json

```diff
@@ -36,40 +20,28 @@
 		"url": "git+https://github.com/bitfocus/companion-module-soundcraft-ui.git"
 	},
 	"license": "MIT",
-	"bugs": {
-		"url": "https://github.com/bitfocus/companion-module-soundcraft-ui/issues"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged"
-		}
-	},
+	"prettier": "@companion-module/tools/.prettierrc.json",
 	"lint-staged": {
-		"*.{js,json,md}": [
-			"prettier --write",
-			"git add"
+		"*.{css,json,md,scss}": [
+			"prettier --write"
 		],
-		"*.{ts}": [
-			"yarn lint --fix --",
-			"git add"
+		"*.{ts,tsx,js,jsx}": [
+			"yarn lint:raw --fix"
 		]
 	},
+	"engines": {
+		"node": "^14 || ^16"
+	},
 	"dependencies": {
-		"@types/node": "^12.0.4",
+		"@companion-module/base": "~0.4.7",
 		"rxjs": "^6.6.3",
 		"soundcraft-ui-connection": "^0.11.0"
 	},
 	"devDependencies": {
-		"@typescript-eslint/eslint-plugin": "~2.31.0",
-		"@typescript-eslint/parser": "~2.31.0",
-		"eslint": "^6.8.0",
-		"eslint-config-prettier": "^6.11.0",
-		"eslint-plugin-prettier": "^3.1.3",
-		"husky": "^3.0.5",
-		"lint-staged": "^9.2.5",
-		"prettier": "^1.17.1",
-		"rimraf": "^3.0.2",
-		"typescript": "~4.0.5"
-	},
-	"homepage": "https://github.com/bitfocus/companion-module-soundcraft-ui#readme"
```

Comment:
I also removed the homepage (it said so in the guide), but this field was just added in 9ca37eb873c537b55d1e a few months back.

Reply:
it was added a few months back so that we could provide some links in the ui to modules. But in v3 those links have moved into `companion/manifest.json`.
It isnt a problem to have it in package.json too, but it serves no purpose there other than conforming to standard package.json layout